velox/functions/prestosql/Arithmetic.h

```diff
@@ -495,5 +495,41 @@ struct TruncateFunction {
   }
 };
 
+template <typename T>
+struct BinomialCDF {
+  // The computation of binomial cumulative distribution follows the algorithm
+  // here: https://mathworld.wolfram.com/BinomialDistribution.html
+
+  double choose(int64_t n, int64_t k) {
+    if (k == 0) {
+      return 1;
+    } else {
+      return (n * choose(n - 1, k - 1)) / k;
+    }
+  }
+
+  FOLLY_ALWAYS_INLINE void
+  call(double& result, int64_t numOfTrials, double successProb, int64_t value) {
+    VELOX_USER_CHECK_GT(
+        numOfTrials, 0, "Number of trials must be greater than 0");
+    VELOX_USER_CHECK_GE(value, 0, "Value must be a positive integer");
```

Comment:
Hmm.... I don't see this and the below check ("Number of trials must be greater than or equal to value") in the Presto java code.  Please can you clarify them.

Reply:
@aditi-pandit Yes this check is missing in Java code, don't know why. Its clearly mentioned in presto binomial_cdf documentation ([binomial_cdf](https://prestodb.io/docs/current/functions/math.html?highlight=binomial#binomial_cdf)) as "numberOfTrials and value must be positive integers with numberOfTrials greater or equal to value".

Reply:
Thanks @gopukrishnasIBM. Did you try Presto SQL queries with such values in the invocation ? Would be interesting to see how Presto behaves. We can then evaluate if we want to add these errors.

Reply:
@aditi-pandit For this check, Java code returns '1.0' if the numberOfTrials is greater or equal to value. Can I implement the same logic here also ?
All other checks behave same.

Reply:
@gopukrishnasIBM : Yes, I think we should make it consistent with Presto.

Reply:
@aditi-pandit Changes made to make it consistent with Presto.